azure-iot-hub/azure/iot/hub/iothub_amqp_client.py

```diff
@@ -70,20 +113,24 @@ def send_message_to_device(self, device_id, message, app_props):
         message = uamqp.Message(
             msg_content, properties=msg_props, application_properties=app_properties
         )
-        self.amqp_client.queue_message(message)
-        results = self.amqp_client.send_all_messages(close_on_done=False)
+        self.amqp_message_send_client.queue_message(message)
```

Comment:
We need to check if the uAMQP object exists and create it if it doesn't.

Reply:
That happens in the `amqp_message_send_client` property above.